config/locales/models/en.yml

```diff
@@ -109,3 +109,5 @@ en:
         user_defined_tags_count: "Fandom, relationship, and character tags"
       work:
         user_defined_tags_count: "Fandom, relationship, character, and additional tags"
+  subscriptions:
+    deleted: "Deleted item"
```

Comment:
Just one more tiny nitpick -- can you add the missing newlines to the end of this file and locales/views/en.yml?